docs/schemas-tests/XR-native-pa-yaml/Src/Components/MyHeaderComponent.pa.yaml

```diff
@@ -0,0 +1,101 @@
+ComponentDefinitions:
+  MyHeaderComponent:
+    CustomProperties:
+      AppTitle:
+        PropertyKind: Input
+        DisplayName: App Title
+        Description: The title of the App
+        DataType: Text
+        Default: ="My App"
+        RaiseOnReset: false
```

Comment:
It probably should have "=" before false

Reply:
So, all of these fields are actually not expressions, other than Default. Seriailzing them as yaml values is the right approach here.

Reply:
The other fields here are just serialized values, not expressions, so no need for `=`.

Reply:
McCall is correct. While the control tree has `Properties` which are mappings from a named property to a Power Fx expression (which must currently start with `=`). But here, only the `Default` property have a value of type Power Fx expression.